tests/system/action/meeting/test_update.py

```diff
@@ -325,6 +325,69 @@ def test_update_topic_poll_default_group(self) -> None:
         self.assert_model_exists("meeting/1", {"topic_poll_default_group_ids": [5]})
         self.assert_model_exists("group/5", {"used_as_topic_poll_default_id": 1})
 
+    def test_update_only_one_time_1(self) -> None:
+        _, response = self.basic_test({"start_time": 150000}, check_200=False)
+        assert (
+            "Only one of start_time and end_time is not allowed."
+            in response.json["message"]
+        )
+
+    def test_update_only_one_time_2(self) -> None:
+        _, response = self.basic_test({"end_time": 156000}, check_200=False)
+        assert (
+            "Only one of start_time and end_time is not allowed."
+            in response.json["message"]
+        )
+
+    def test_update_only_one_time_one_removal_from_db(self) -> None:
+        self.set_models(
+            {
+                "committee/1": {"name": "test_committee"},
+                "group/1": {},
+                "meeting/1": {
+                    "name": "test_name",
+                    "is_active_in_organization_id": 1,
+                    "committee_id": 1,
+                    "default_group_id": 1,
+                    "projector_ids": [1],
+                    "reference_projector_id": 1,
+                    "default_projector_$_ids": Meeting.default_projector__ids.replacement_enum,
+                    **{
+                        f"default_projector_${name}_ids": [1]
+                        for name in cast(
+                            List[str], Meeting.default_projector__ids.replacement_enum
+                        )
+                    },
```

Comment:
Are the projector fields really needed for this test?

Reply:
No, not really, I removed some stuff from the test.